py-kms/pykms_WinService.py

```diff
@@ -0,0 +1,55 @@
+# Warning: this script only for Windows 7 SP1 / Windows 2008 R2!
+# -
```

Comment:
Code cannot contain installation instructions.

Reply:
This is just the Windows Service right? These instruction should go to the docs anyways.